app/controllers/api_controller/vms.rb

```diff
@@ -150,6 +150,19 @@ def retire_resource_vms(type, id = nil, data = nil)
       end
     end
 
+    def restart_resource_vms(type, id = nil, _data = nil)
```

Comment:
change API action same as model method, so "reboot_guest", updating api.yml and spec accordingly.  Thanks.